### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by quanvo87**

### CODE REVIEW

File: Example/Tests/Tests.swift [hunk 1]
```diff
@@ -1,29 +1,138 @@
...
+//class Tests: XCTestCase {
+//    func testDecrement() {
+//        let expectation = self.expectation(description: #function)
+//        defer { waitForExpectations(timeout: 10) }
+//
+//        let rand1 = Int(arc4random_uniform(500))
+//        let rand2 = Int(arc4random_uniform(500))
+//        FirebaseHelper.main.set(rand1, at: "test") { (error) in
+//            XCTAssertNil(error, String(describing: error))
+//            let work = GroupWork()
```
Comment (dingwilson) [inline Example/Tests/Tests.swift:13]: Am I missing something? It doesn't seem like you've imported GroupWork anywhere.. Haven't tried yet, but I don't think the test would compile correctly.

File: README.md [hunk 2]
```diff
@@ -5,12 +5,10 @@
 [![License](https://img.shields.io/cocoapods/l/FirebaseHelper.svg?style=flat)](http://cocoapods.org/pods/FirebaseHelper)
 [![Platform](https://img.shields.io/cocoapods/p/FirebaseHelper.svg?style=flat)](http://cocoapods.org/pods/FirebaseHelper)
```
Comment (dingwilson) [inline README.md:7]: Here, I suggest adding a brief introduction (and maybe some bullets) about what exactly FirebaseHelper is, and what it can do. Maybe something like: 

> FirebaseHelper is a small wrapper over Firebase's realtime database, providing streamlined methods to get, set, delete, and increment values.
>
> ## Features
> - [x] Setup Firebase Realtime Database Ref
> - [x] Read values (get)
> - [x] Write/Update values (set)
> - [x] Remove values (delete)
> - [x] Increment values (increment)
> - [ ] -insert future functionality here-

File: README.md [hunk 3]
```diff
@@ -20,9 +18,127 @@ it, simply add the following line to your Podfile:
...
+  }
+}
+
+FirebaseHelper.main.delete("users", "john123", "favoriteFood") { error in
+    if let error = error {
+      // handle error
+  }
+}
+
+FirebaseHelper.main.increment(5, "users", "john123", "favoriteFoodEatenThisMonth") {
```
Comment (dingwilson) [inline README.md]: This function confuses me:

Is 5 the value favoriteFoodEatenThisMonth is being "increment"ed to? Or will the value be "increment"ed to 6 (5++)? Also why is this necessary? (how is this different from `.set`)

**Review by dingwilson — COMMENTED** [review]
Looks great overall! I did have 2 questions, and also one small suggestion...

File: Example/Tests/Tests.swift (hunk 1 above)
Comment (quanvo87) [reply Example/Tests/Tests.swift:13]: The tests are just placeholder...I think I'll save adding tests for later. I think the correct way to do them would be:

- create a private repo with a firebase plist for a separate firebase project
- write a travis script that:
  - downloads the private repo with credentials in travis env vars
  - copies over the plist into this project somewhere
  - then the tests can login to the database and run

Not even sure that would work. Maybe there's an easier way...

File: README.md (hunk 2 above)
Comment (quanvo87) [reply README.md:7]: Good idea! I'll work on this.

File: README.md (hunk 3 above)
Comment (quanvo87) [reply README.md]: This is confusing. It will increment _by_ 5. I'll add that parameter name for clarity.

[Firebase docs](https://firebase.google.com/docs/database/ios/read-and-write#save_data_as_transactions)

This is necessary because one cannot simply say `set(oldValue + 5)` because of race conditions. Say concurrent `set()`s are taking place, `oldValue`s can go stale before `oldValue + 5`s take place, etc. Thus this API is a wrapper around Firebase's solution for this problem.

File: Example/Tests/Tests.swift (hunk 1 above)
Comment (dingwilson) [reply Example/Tests/Tests.swift:13]: ahh I get it.. You're having issues because you're trying to test against/hit Firebase. Yeah, what you're suggesting makes sense to me.

File: README.md (hunk 3 above)
Comment (dingwilson) [reply README.md]: ahhh okay that makes sense. Now that I think about it, thats actually extremely useful! Thanks for the explanation.

**Review by dingwilson — APPROVED** [review]
LGTM
